zproject/prod_settings_template.py

```diff
@@ -254,6 +254,9 @@
         # the left end of the login/register buttons for this IDP.
         # The default of None results in a text-only button.
         # "display_icon": "/path/to/icon.png",
+        
+        # Activate checking entitlements for strict SAML authentication control
+        # "requiredEntitlements": ["urn:oid:1.3.6.1.4.1.5923.1.1.1.9"],
```

Comment:
Is `requiredEntitlements` a standard name used in SAML elsewhere?  IF not, we should call this, `required_entitlements`, for consistency.  Or if the intent is that this will only be related to `OID_EDU_PERSON_ENTITLEMENTS`, maybe we should call this `required_oid_edu_person_entitlements`.

Reply:
@timabbott as i say there is so little information about that on python-saml page. I did not test, but i will. I guess it will be not a problem changing it to `required_entitlements`